bundles/org.openhab.binding.ojelectronics/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,165 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" bindingId="ojelectronics" xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+	<!-- Bridge -->
+	<bridge-type id="bridge">
+		<label>OJ Electronics Bridge</label>
+		<description>Bridge to access OJ Electronic devices.</description>
+		<config-description>
+			<parameter name="userName" type="text" required="true">
+				<label>User Name</label>
+				<description>User Name for access to OJ Electronics cloud service.</description>
+			</parameter>
+			<parameter name="password" type="text" required="true">
+				<label>Password</label>
+				<description>Password for access to OJ Electronics cloud service.</description>
+				<context>password</context>
+			</parameter>
+			<parameter name="apiUrl" type="text" required="true">
+				<label>API-URL</label>
+				<description>URL to cloud API-service.</description>
+				<context>url</context>
+				<advanced>true</advanced>
+				<default>https://OWD5-OJ001-App.ojelectronics.com/api</default>
+			</parameter>
+			<parameter name="refreshDelayInSeconds" type="integer" required="true" min="15">
+				<label>Refresh delay</label>
+				<description>Refresh delay in seconds.</description>
+				<advanced>true</advanced>
+				<default>30</default>
+			</parameter>
+			<parameter name="customerId" type="integer" required="true">
+				<label>Customer ID</label>
+				<description>Customer ID</description>
+				<advanced>true</advanced>
+				<default>1</default>
+			</parameter>
+			<parameter name="APIKEY" type="text" required="true">
+				<label>APIKEY</label>
+				<description>APIKEY</description>
+				<advanced>true</advanced>
+				<context>password</context>
+				<default>f219aab4-9ac0-4343-8422-b72203e2fac9</default>
+			</parameter>
+			<parameter name="softwareVersion" type="integer" required="true">
```

Comment:
Should this be configureable by the user or is it more a constant, that depends on your current implementation?